### PR TITLE
Read me Error fixing and warning changes

### PR DESCRIPTION
**Pull request by sandeepjak2007**
Error in Read me file and some warning removed

### CODE REVIEW

**Comment by sandeepjak2007** [comment]
Can anyone suggest how to resolve this issue. And what I did wrong in this commit

**Comment by IndusAryan** [comment]
nothing wrong, ignore it

File: README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@
 [![Lemmy](https://libre-tube.github.io/assets/widgets/lemmy-widget.svg)](https://feddit.rocks/c/libretube)
 
 </div>
-
-> **Note** <br>
-> We don't accept feature or bug requests on these platforms. Kindly submit requests only on GitHub.
+<div>
+**Note** <br>
```
Comment (Bnyro) [inline README.md:13]: This change removes the bold formatting of "Note".

**Review by Bnyro — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sandeepjak2007) [reply README.md]: @Bnyro Can you please check code once again and review it

**Comment by Bnyro** [comment]
It's still not bold for some reason (see the github preview of the readme at your branch)

**Comment by sandeepjak2007** [comment]
> It's still not bold for some reason (see the github preview of the readme at your branch)

Can you please check once

**Comment by Bnyro** [comment]
Sorry, I don't really see the point of this PR (and you don't even seem to be willing to test your changes)...

**Pull request closed without merging**